AsBuiltReport.VMware.vROps.json

```diff
@@ -6,22 +6,23 @@
     },
     "Options": {
         "AuthSource": "local",
-        "AlertFilter": "*"
+        "AlertFilter": "*",
+        "AlertSeverityFilter": ["Critical", "Immediate", "Warning", "Information", "Auto"]
     },
     "HealthCheck": {},
     "InfoLevel": {
-        "_comment_": "0 = Disabled, 1 = Summary, 2 = Informative, 3 = Detailed",
-        "GlobalSettings": 3,
-        "Authentication": 3,
-        "Roles": 3,
-        "Groups": 3,
-        "Users": 3,
-        "Adapters": 3,
-        "RemoteCollectors": 3,
-        "Alerts": 3,
-        "Symptoms": 3,
-        "SuperMetrics": 3,
-        "ServiceStatus": 3,
-        "CustomGroups": 3
+        "_comment_": "0 = Disabled, 1 = Informative, 2 = Comprehensive",
```

Comment:
InfoLevel naming does not align to README

Reply:
Updated README